lightning/src/util/config.rs

```diff
@@ -126,6 +126,30 @@ pub struct ChannelHandshakeConfig {
 	///
 	/// [`KeysInterface::get_shutdown_scriptpubkey`]: crate::chain::keysinterface::KeysInterface::get_shutdown_scriptpubkey
 	pub commit_upfront_shutdown_pubkey: bool,
+
+	/// `their_channel_reserve_proportion_millionth` is proportion of the channel_value to configure
```

Comment:
nit: No need to repeat the field name here. It's also probably generally advisable to reduce the use of variable notation in the public API documentation, if you do not mean to link something specific. So instead of, for example, `channel_value`, just write "value of the channel".
```suggestion
	/// The proportion of the channel value to configure as the
```